spec.md

```diff
@@ -118,6 +118,7 @@ of the delivery domain) can perform such a downgrade or interception attack.
 This document defines a mechanism for recipient domains to publish policies
 specifying:
 
+   * MTA that handles emails for the recipient domain
```

Comment:
I wouldn't bother to call this out (the "MTA that handles email for the recipient domain") because that's really what we mean by the third bullet point (how to validate the TLS server certificate). What do you think?

(Note I'm making a few edits to your pull request in another branch, so just commenting here so you're aware of my reasoning in case you disagree.)